features/alert_history.feature

```diff
@@ -38,15 +38,15 @@ Feature: Alert history reports
 	@configuration
 	Scenario: Single host alert history
 		Given I visit the alert history page for host "linux-server1"
-		Then I should see "OK - Sven Melander"
-		And I should see "Reporting period: Forever"
+		Then I should see "Reporting period: Today"
+		And I should see "No log data recorded during this time"
```

Comment:
This makes it so that we don't have a test that checks if we have a data entry in the default view
Better if we can add a table entry that has today's date. 
Maybe you can create a step definition similar to 
monitor-ninja/features/step_definitions/nagios.rb:21

That uses todays date instead of hard coded date